Examples/Python/python/acts/examples/reconstruction.py

```
@@ -140,33 +140,38 @@
 )
 
 
-def trackSelectorDefaultKWArgs(c):
+def trackSelectorDefaultKWArgs(c, overwriteArgs=dict()):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)